docker-hub/registry-access-management.md

```diff
@@ -0,0 +1,114 @@
+---
+description: Registry Access Management
+keywords: registry, access, managment
+title: Registry Access Management
+---
+
+Registry Access Management is a new feature available to organizations with a Docker Business subscription. This feature lets organization owners manage the registries that their users can access while using Docker Desktop.  When using this feature, organization owners can ensure that their users can only access their trusted registries, such as a secure private registry on Artifactory, thereby reducing the security risks that can occur when users interact with public registries.
```

Comment:
Do we want to say that it's "new" within the text? Will we remember to remove that in a few months time? I would just delete "a new feature" and say "is available to". But you're the expert so I'm happy to be overruled.

Reply:
Thanks Stephen! Good point, I removed  "new"